forge-gui/res/cardsfolder/upcoming/hamlet_glutton.txt

```diff
@@ -0,0 +1,13 @@
+Name:Hamlet Glutton
+ManaCost:5 G G
+Types:Creature Giant
+PT:6/6
+K:Bargain
+K:Trample
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$This spell costs {2} less to cast if it's bargained.
```

Comment:
check with ValidSpell$ and hardcode Amount

Reply:
current version not working

Reply:
Try `Spell.Bargained`

Reply:
thanks seems to work